docs/pages/includes/database-access/db-configure-start.mdx

```diff
@@ -0,0 +1,127 @@
+{{ dbName="test" }}
+<Tabs>
+<TabItem label="Using a config file">
+On the host where you will run the Teleport Database Service, start Teleport
+with the appropriate configuration.
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=teleport.example.com:443 \
+   --name={{ dbName }} \
+   --protocol={{ dbProtocol }} \
+   --uri={{ databaseAddress }} \
+   --labels=env=dev 
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=mytenant.teleport.sh:443 \
+   --name={{ dbName }} \
+   --protocol={{ dbProtocol }} \
+   --uri={{ databaseAddress }} \
+   --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  A single Teleport process can run multiple different services, for example
+  multiple Database Service agents as well as the SSH Service or Application
+  Service.
+
+</Admonition>
```

Comment:
Instead of having this as a callout under the command, what about describing it inline above, and explaining that this command would replacing an existing configuration file, with steps to mitigate?

Reply:
This is an across the board issue I think.  Seems like something we need to come back to with a common solution. As this supports helm and linux machines not sure we can present something quickly here.

Reply:
Provided my version of a solve as 9166c63c277dcfc19c7a0cba4dabe38780d018dc